app/assets/stylesheets/custom.scss

```
@@ -0,0 +1,93 @@
+@import "bootstrap-sprockets";
+@import "bootstrap";
+
+/* universal */
+body {
+padding-top: 60px;
```

```suggestion
   padding-top: 60px;
```
sai indent hết r
e fix nhé. tab 2space